### ISSUE TITLE
[FEATURE] Hide others' funding until finish

### ISSUE THREAD

**Issue by alanna** [cobudget/cobudget#625]
This is a suggestion for an optional setting where funders can't see other people's funding choices until the end of the round or until the person has spent all their funds, so people think more independently and aren't influenced by groupthink. The user should still be able to see their own funding choices though. This is not always what's right for a group, but sometimes it's a useful option. This was requested by a user in our group's recent round.

**Comment by aerugo** [comment]
I fully agree with this idea @alanna. This is needed for the Borderland too, and it was the default behaviour on the Dreams platform (a precursor to the current version of Cobudget).

**Comment by aerugo** [comment]
@alanna, could I get a clarification here:

Did you mean to hide the list of contributors until the end of the round, or did you mean to not show the progress per bucket at all? What is important - hiding _who_ has contributed, hiding _how much_ has been contributed, or both?

**Comment by alanna** [comment]
The way I was imagining it, both would be hidden, as both can influence your thinking.
